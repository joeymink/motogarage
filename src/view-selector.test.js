import React from 'react';
import ViewSelector from './view-selector';
import renderer from 'react-test-renderer';

test('ViewSelector renders as expected', () => {
  const component = renderer.create(
    <ViewSelector />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
