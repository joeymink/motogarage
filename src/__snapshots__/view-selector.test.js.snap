exports[`test ViewSelector renders as expected 1`] = `
<div>
  <button
    className="btn btn-default"
    disabled={false}
    onClick={[Function]}
    type="button">
    left
  </button>
  <button
    className="btn btn-default"
    disabled={false}
    onClick={[Function]}
    type="button">
    right
  </button>
</div>
`;
